i64.js

```diff
@@ -128,8 +127,32 @@ I64LongRW.prototype.readFrom = function readFrom(buffer, offset) {
 
 var i64LongRW = new I64LongRW();
 
-function I64BufferRW() {
-}
+function I64DateRW() {}
+
+util.inherits(I64DateRW, I64RW);
+
+I64DateRW.prototype.readFrom = function readFrom(buffer, offset) {
+    var long = Long.fromBits(
+        buffer.readInt32BE(offset + 4, 4, true),
+        buffer.readInt32BE(offset + 0, 4, true)
+    );
+    var seconds = long.divide(1000).toNumber();
+    var value = new Date(seconds);
+    return new bufrw.ReadResult(null, offset + 8, value);
+};
+
+I64DateRW.prototype.writeInto = function writeInto(value, buffer, offset) {
+    var self = this;
+    if (typeof value === 'string') {
+        value = Date.parse(value);
```

Comment:
I will have a talk with my brain later about this. I have addressed the issue by getting rid of the scale machinations.